docs/sources/k6/next/set-up/set-up-distributed-k6/usage/istio.md

```diff
@@ -0,0 +1,41 @@
+---
+weight: 500
+title: Use the k6 Operator with Istio
+---
+
+# Use the k6 Operator with Istio
+
+To use the k6 Operator against a cluster with [Istio](https://istio.io) support, you may need to perform some additional configuration. At the time of writing, Istio has two modes of execution: sidecar mode and ambient mode.
+
+## Sidecar mode
+
+If you use Istio in the [sidecar mode](https://istio.io/latest/docs/setup/), firstly, check the versions of your cluster and Istio. For the k6 Operator to work without issues with Istio in the sidecar mode, Istio's version must be >= 1.19 and Kubernetes version must be >= 1.28. 
+
+Additionally, Kubernetes cluster must have native sidecars [enabled](https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/), which is the default behaviour since version 1.29 but must be enabled with a feature gate in version 1.28. Here's an example of the [kind cluster](https://kind.sigs.k8s.io/) with this configuration:
+
+```yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+featureGates:
+  SidecarContainers: true
+```
+
+If the above requirements are not fulfilled, the k6 Operator will not be able to work with Istio because of [this issue](https://github.com/istio/istio/issues/11045): the non-native sidecar `istio-proxy` does not exit and all Kubernetes Jobs end up stuck, including the ones started by the k6 Operator. Because of this, k6 Jobs enter a `NotReady` state, and the k6 Operator does not know how to handle it in a general case.
```

Comment:
```suggestion
If the above requirements are not fulfilled, k6 Operator will not be able to work with Istio because of [this issue](https://github.com/istio/istio/issues/11045): the non-native sidecar `istio-proxy` doesn't exit and all Kubernetes Jobs end up stuck, including the ones started by the k6 Operator. Because of this, k6 Jobs enter a `NotReady` state, and the k6 Operator doesn't know how to handle it in a general case.
```